src/v2/query/queryBuilderV2.ts

```diff
@@ -651,23 +651,40 @@ export class QueryBuilderV2 {
     }
     let valid = true;
 
-    // Check if callback address is a valid contract address
-    const bytecode = await this.config.provider.getCode(this.callback.target);
-    if (bytecode.length <= 2) {
-      console.warn("Callback address is not a valid contract address");
-      valid = false;
+    let target = this.callback.target;
+    if (
+      !(
+        target === undefined || 
+        target === "" || 
+        target === ethers.ZeroAddress
```

Comment:
It seems like this should make the query invalid -- is that covered somewhere else?